compiler/fir/tree/src/org/jetbrains/kotlin/fir/FirRenderer.kt

```diff
@@ -923,6 +923,12 @@ class FirRenderer(builder: StringBuilder) : FirVisitorVoid() {
         }
     }
 
+    override fun visitCheckNotNullCall(checkNotNullCall: FirCheckNotNullCall) {
+        checkNotNullCall.annotations.renderAnnotations()
+        print("!!")
```

Comment:
I think, it's better to swap this and next lines, because of `expression!!` is more readable than `!!expression`

Reply:
I agree. I also eliminated the surrounding parentheses from calling `visitCall`, by directly rendering the argument.